tests/epyccel/test_epyccel_dicts.py

```diff
@@ -181,3 +181,15 @@ def get_default_str_keys():
     python_result = get_default_str_keys()
     assert isinstance(python_result, type(pyccel_result))
     assert python_result == pyccel_result
+
+def test_dict_clear(language):
+    def dict_clear():
+        a = {1:1.0, 2:2.0}
+        a.clear()
+        return a
```

Comment:
You will need to use `len` to test in order to be able to test other languages (C/Fortran) as returning a dictionary is not yet supported for these languages
```suggestion
        a.clear()
        return len(a)
```